pages/linux/cjxl.md

```diff
@@ -0,0 +1,16 @@
+# cjxl
+
+> Compress images to JPEG XL.
```

Comment:
```suggestion
> Compress images to JPEG XL.
> Accepted input formats are PNG, APNG, GIF, JPEG, EXR, PPM, PFM, PAM, PGX, or JXL.
```

```suggestion
> Compress images to JPEG XL.
> Accepted input extensions are PNG, APNG, GIF, JPEG, EXR, PPM, PFM, PAM, PGX, or JXL.
```

Reply:
"extensions" is probably better since we're using `.ext` in a command.